cl/search/tests.py

```diff
@@ -3627,7 +3649,7 @@ def test_oa_search_by_no_exact_docket_number(self) -> None:
         # Frontend
         search_params = {
             "type": SEARCH_TYPES.ORAL_ARGUMENT,
-            "q": f'docketNumber:"19 5734"',
+            "q": f'docketNumber:"19 5734" OR docketNumber:19:5734',
```

Comment:
How does this change things? It's unclear why there's an `OR` here, no?

Reply:
Yeah, you're right, the second query in this test has no effect.

I wanted to test `docketNumber:19:5734` works properly since `:` break queries like these. We have code in `build_fulltext_query` to handle this issue escaping `:` in `docketNumber` queries.

So I removed the `OR docketNumber:19:5734` and added a test for this specific case underneath: `docketNumber:19:5734`

Thanks, let me know if there is anything else here.

Reply:
That's it, thanks.